flexget/plugins/input/ftp_list.py

```diff
@@ -143,8 +148,7 @@ def _handle_path(self, entries, ftp, baseurl, path='', mlst_supported=False, fil
                                   recursive, get_size, encoding)
 
             if not files_only or mlst.get('type') == 'file':
-                url = baseurl + path + '/' + p
-                url = url.replace(' ', '%20')
+                url = baseurl + urllib.quote_plus(path) + '/' + urllib.quote_plus(p)
```

Comment:
It doesn't. Here's the python documentation: https://docs.python.org/2/library/urllib.html
As you can see the path separator is considered one of the default "safe" characters that does not get converted.